sycl/include/CL/sycl/handler.hpp

```diff
@@ -386,7 +386,19 @@ class __SYCL_EXPORT handler {
                        static_cast<int>(AccessTarget), ArgIndex);
   }
 
-  template <typename T> void setArgHelper(int ArgIndex, T &&Arg) {
+  template <typename T> struct ShouldEnableSetArgHelper {
+    static constexpr bool value = std::is_trivially_copyable<T>::value
+#ifdef CL_SYCL_LANGUAGE_VERSION
+#if CL_SYCL_LANGUAGE_VERSION <= 121
```

Comment:
BTW, I'm not sure about these ifdefs.
AFAIK, currently our compiler defines CL_SYCL_LANGUAGE_VERSION to be equal to 121 (since 1.2.1 is only supported).
https://github.com/intel/llvm/blob/4968e7c54d5030ee60b87efcfdfb21e6a7f0cfa6/clang/lib/Frontend/InitPreprocessor.cpp#L467
At the same time, SYCL 2020 defines new macro SYCL_LANGUAGE_VERSION. (5.6 Preprocessor directives and macros)

So, will the compiler define only SYCL_LANGUAGE_VERSION or it'll define two macros with the same value for SYCL-2020?
@bader, what do you think? Should I simplify these lines to this?
```suggestion
#if CL_SYCL_LANGUAGE_VERSION == 121
```

Reply:
This version looks okay to me.